src/category_theory/limits/shapes/terminal.lean

```diff
@@ -204,10 +216,90 @@ def initial_unop_of_terminal {X : Cᵒᵖ} (t : is_terminal X) : is_initial X.un
 { desc := λ s, (t.from (opposite.op s.X)).unop,
   uniq' := λ s m w, quiver.hom.op_inj (t.hom_ext _ _) }
 
+/-- A category is a `zero_le_category` if the canonical morphism of an initial object is a
+monomorphism.
+In practice, this is most useful when given an arbitrary morphism out of the chosen initial object,
+see `initial.mono_from`.
+
+TODO: This is a condition satisfied by categories with zero objects and morphisms, as well as
+categories with a strict initial object; though these conditions are essentially mutually exclusive.
+-/
+class zero_le_category (C : Type u) [category.{v} C] : Prop :=
```

Comment:
What do you think of `bot_le_category`?

Reply:
I think it's definitely a step up! To be honest I'm not sure I like the `_category` suffix, there aren't any other category properties where we use this, but also `bot_le` on its own is already used...

Reply:
Hmm, so `category.bot_le` or `category_theory.bot_le`? Not sure if those are really better...

Reply:
And `initial_mono_class` is maybe also a bit verbose.

Reply:
`initial_mono_class` works for me!